mozilla-release/browser/components/newtab/aboutNewTabService.js

```diff
@@ -98,6 +98,11 @@ AboutNewTabService.prototype = {
   _overridden: false,
   willNotifyUser: false,
 
+  CLIQZ_NEW_TAB_URLS: [
+    "resource://cliqz/freshtab/home.html",
+    "about:cliqz"
+  ],
+
```

Comment:
@chrmod is it right?

Reply:
looks good to me. we should definitely use resource:// url fro freshtab.